specreduce/wavelength_calibration.py

```diff
@@ -209,6 +213,9 @@ def wcs(self):
         # Fit the model
         self._model = fitter(self._model, x, y)
 
+        # Get the fit residuals by evaulating model
+        self._fit_resids = y - self._model(x)
```

Comment:
I wonder if this should just be its own cached property so that it knows to clear when the model is manually changed by the user and to save a (small) bit of time when applying to a spectrum but not requesting residuals?

Reply:
done